roles/edpm_nova/defaults/main.yml

```diff
@@ -27,6 +27,7 @@ edpm_nova_image_download_retries: 5
 # dest dir is on the target host
 edpm_nova_config_src: /var/lib/openstack/configs
 edpm_nova_config_dest: /var/lib/openstack/config/nova
+edpm_nova_config_sync_dest: /tmp/nova_sync/
```

Comment:
I guess we need an argspec for this as the rest has it.

Reply:
Alternatively we could use ansible.builtin.tempfile module

Reply:
done